_pages/security/scanning.md

```diff
@@ -21,3 +21,41 @@ Alerts come in one of four categorizations:
 * **High** and **Critical** must be resolved before the ATO is granted.
 
 Any false positives should be documented as such, through an "ignore" file (e.g. with a `note` in [Brakeman](http://brakemanscanner.org/docs/ignoring_false_positives/)) or with your ATO-related documentation.
+
+## ATO Scanning 
+
+### Getting ready to scan
+
+The system's technical stack needs to be relatively stable before authorization.
+
+If, during testing, the system performs:
+
+* user authentication or authorization
+* back-end administrative functions
+* encryption
+
+...then those features cannot be "relatively simple" but must in fact be *complete* before an authorization will be given. Note that the use of common web frameworks and [18F HTTPS standards](https://github.com/18F/https) resolves these issues in almost every case.
+
+Lastly, make sure the `README` file in your repo is fully up to date and clearly explains what the system does and why at a high level. You should also include the above information in your `README`.
```

Comment:
That's why I had these three sections under ATO on the testing page. I think they belong together as part of the narrative: Getting ready to scan -> Greybox testing -> Expectation management. 

That narrative could live here or under the ATO topic. The ATO page itself is pretty minimal, it doesn't make a whole lot of sense to add this step to that page, when we have the rest of the ATO info spread across 8 sub-pages.